core/src/main/java/org/hibernate/ogm/storedprocedure/impl/NoSQLProcedureOutputsImpl.java

```diff
@@ -54,6 +54,13 @@ public NoSQLProcedureOutputsImpl(NoSQLProcedureCallImpl procedureCall) {
 		this.procedureCall = procedureCall;
 		this.sessionFactory = procedureCall.getSession().getFactory();
 		this.gridDialect = this.sessionFactory.getServiceRegistry().getService( StoredProcedureAwareGridDialect.class );
+		//TODO: Check this "if"
```

Comment:
I think we could remove TODO. If the service was not available gridDialect sould be null, from what I tested.

Reply:
thank you, I will remove soon